Do not try to build the libev subdirectory; we are using MacPorts libev.

Fixes:

make[2]: *** No rule to make target `all'.  Stop.
--- Makefile.am.orig	2020-09-09 09:42:01.000000000 -0500
+++ Makefile.am	2024-01-22 23:58:56.000000000 -0600
@@ -1,5 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4 -I libmissing/m4
-SUBDIRS = docs libev libmissing m4 plugins src
+SUBDIRS = docs libmissing m4 plugins src
 
 EXTRA_DIST = AUTHORS COPYING HACKING.README INSTALL NEWS README \
              selinux/prelude-manager.te \
